### ISSUE TITLE
grid is not responsive

### ISSUE THREAD

**Issue by ginetsl** [plone/plone.app.mosaic#299]
The mosaic grid is not responsive. Tiles don't stack on mobile devices.

To reproduce, put 2-3 Rich Text tiles side by side in a row and test in mobile screen size.

Plone = 5.0.6
Theme = Barceloneta
plone.tiles = 1.8.0
plone.app.tiles = 3.0.0
plone.app.standardtiles = 2.0.0rc1
plone.app.blocks = 4.0.0rc1
plone.app.drafts = 1.1.1
plone.app.mosaic = 2.0.0rc1

**Comment by datakurre** [comment]
Can this be reproduced with just Plone 5 default theme?

**Comment by ginetsl** [comment]
Do you mean without using plone.mosaic?

**Comment by datakurre** [comment]
I mean, on a new Plone 5 site with just Mosaic added.

**Comment by ginetsl** [comment]
Yes, the same happens with just mosaic installed in a Plone 5 site using the default theme (Barceloneta)

**Comment by agitator** [comment]
Also have that problem, seems the mosaic selectors we're injecting are too narrow compared to bootstrap default

**Comment by datakurre** [comment]
@agitator As an author or that change, could you fix it?

**Comment by agitator** [comment]
yes, have to do it for my client anyway ;-)
